packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -34,16 +34,25 @@ export interface FileUploaderProps {
   variation?: 'drop' | 'button';
 }
 
+export interface UploadIconProps {
```

Comment:
Can we either align on `Upload` (preferred) or `Uploader`?

Reply:
I can rename to `Upload` on the next PR